sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java

```diff
@@ -122,7 +122,7 @@ public KeyVaultKeyStore() {
         String clientId = System.getProperty("azure.keyvault.client-id");
         String clientSecret = System.getProperty("azure.keyvault.client-secret");
         String managedIdentity = System.getProperty("azure.keyvault.managed-identity");
-        long refreshInterval = Optional.of("azure.keyvault.jca.certificates-refresh-interval")
+        long refreshInterval = Optional.of("azure.keyvault.jca.certificates-refresh-interval-in-ms")
```

Comment:
How about not introduce breaking change by doing like this:
```
Stream.of("azure.keyvault.jca.certificates-refresh-interval-in-ms", "azure.keyvault.jca.certificates-refresh-interval")
.map(System::getProperty)
.map(Long::valueOf)
.filter(Object::nonNull)
.findFirst()
.orElse(0L)
```

And write unit test for this logic.